apps/website-25/src/components/courses/exercises/FreeTextResponse.test.tsx

```diff
@@ -1,10 +1,24 @@
-import { render, waitFor } from '@testing-library/react';
-import { describe, expect, test } from 'vitest';
+import { render } from '@testing-library/react';
+import {
+  describe,
+  expect,
+  Mock,
+  test,
+  vi,
+} from 'vitest';
+import axios from 'axios';
 import FreeTextResponse from './FreeTextResponse';
 
+// Mock axios
+vi.mock('axios');
```

Comment:
I think because we've now moved all the networking logic out of here (yay) this isn't necessary?